apps/vaccines-updates-client/src/services/vacgaps-api-client.ts

```diff
@@ -18,7 +18,7 @@ export async function sendReport(report: VaccinesReport, facebookAccessToken: st
 
 export async function isSupervisor(facebookAccessToken: string): Promise<boolean> {
```

Comment:
[Functional] You check for "true" response from server, but server responds with "True" for some reason.
You can just compare case insensitively for now to avoid this problem.